controllers/logging_controller.go

```diff
@@ -263,8 +263,14 @@ func (r *LoggingReconciler) GetResources(logging *loggingv1beta1.Logging) (*mode
 	loggingResources := model.NewLoggingResources(logging, r.Client, r.Log)
 	var err error
 
+	listOptsForClusterResources := []client.ListOption{}
```

Comment:
```suggestion
	var listOptsForClusterResources []client.ListOption
```
Could save an unnecessary allocation.